calendar-webapp/src/main/java/org/exoplatform/calendar/webui/popup/UIEventAttenderTab.java

```diff
@@ -125,6 +125,10 @@ public static String getFullname(String username) throws Exception {
     User u = CalendarUtils.getOrganizationService().getUserHandler().findUserByName(username);
     if (u == null) return null;
     String fullName = u.getDisplayName();
+    //--- patch to fix ORG-1812

```

Comment:
Comments must be self-explanatory. Describe what it does instead of referencing a jira issue.